frontend/packages/knative-plugin/src/types.ts

```diff
@@ -197,3 +197,24 @@ export enum DomainMappingResponseAction {
 export enum ServerlessBuildStrategyType {
   ServerlessFunction = 'ServerlessFunction',
 }
+
+/* Test Serverless Function */
+export enum InvokeFormat {
+  HTTP = 'HTTP',
+  CloudEvent = 'CloudEvent',
+}
+
+export enum HTTPMessageTypes {
+  Request = 'Request',
+  Response = 'Response',
+}
+
+export enum RequestPaneTab {
+  Body = 'Body',
+  Options = 'Options',
+}
+
+export enum ResponsePaneTab {
+  Body = 'Body',
+  Info = 'Info',
+}
```

Comment:
I think we should keep the focus in the src/types.ts file on the official knative type definitions and move this into a types.ts file in the test-function folder.